Gemfile

```
@@ -41,6 +41,7 @@ gem "tzinfo-data", platforms: %i(mingw mswin x64_mingw jruby)
 
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", require: false
+gem 'bcrypt', '3.1.18'
```

chú ý convention dùng nháy " hết cho file này nhé
```suggestion
gem "bcrypt", "3.1.18"
```